components/wso2is7.key.manager/src/main/java/org/wso2/is7/client/WSO2IS7ConnectorConfiguration.java

```diff
@@ -75,6 +76,84 @@ public List<ConfigurationDto> getConnectionConfigurations() {
         return configurationDtoList;
     }
 
+    public List<ConfigurationDto> getAdvancedConnectionConfigurations() {
```

Comment:
rename this back to auth configs and remove endpoint info from this. get a merge in the UI